providers/hubspot/metadata.go

```diff
@@ -300,16 +301,23 @@ func (f fieldDescription) transformToFieldMetadata() common.FieldMetadata {
 	}
 }
 
-func (f fieldDescription) implyEnumerationType() (common.ValueType, []common.FieldValue) {
+func (f fieldDescription) implyEnumerationType(fieldName string) (common.ValueType, []common.FieldValue) {
 	var values []common.FieldValue
 
 	if len(f.Options) != 0 {
 		// List of values is not nil, at least one option exists.
 		values = make([]common.FieldValue, len(f.Options))
+
 		for index, option := range f.Options {
+			displayValue := option.Label
+			// For persona field, use description if it exists, otherwise fall back to label
+			if fieldName == "hs_persona" && option.Description != "" {
```

Comment:
Good to add context: https://community.hubspot.com/t5/APIs-Integrations/Getting-Wrong-Value-from-Persona-in-API/m-p/1193587/highlight/true#M84004

Reply:
+1 please add this in a comment